src/Tuura/Boolean/Options.hs

```diff
@@ -1,4 +1,4 @@
-module Tuura.Plato.BoolToConcept.Options (Options(..), getOptions) where
+module Tuura.Boolean.Options (Options(..), getOptions) where
```

Comment:
There is separate `Options`, one for `translate/Main.hs` and one for `boolToConcepts/Main.hs`. See the root directory.